### DISCUSSION TITLE
Support other frameworks

### DISCUSSION THREAD

**Discussion by Olyno** [family/connectkit#49] [Ideas]
Hi 👋🏻 

I'm someone who doesn't like coding with React. Would it be possible to have your component in vanilla js, or that supports other frameworks such as Vue, Svelte and Angular?

The best way to do this would be to transform your code into a web component, supporting all existing frameworks on the market.

**Comment by sameoldlab** [comment]
I mainly use Svelte, but agree with @Olyno that doing this with web components would be better than keeping track framework popularity. Perhaps someone from the team (after holidays) would have a better idea  of what can or cannot be converted. If most of the UI and interactions can be created in a web component, then it'd also be easier for anyone to swap in/out the framework dependent state management sections.

**Comment by zk118** [comment]
+1 Would love to integrate it to Vuejs

**Comment by unicape** [comment]
There is already a Vue version of wagmi here
https://github.com/unicape/use-wagmi

**Reply by zk118** [reply]
This discussion is about connectkit, not wagmi

**Comment by DigitalOutbreak** [comment]
would love a svelte version of this.
